components/ColorPicker/src/Converters/NullToTransparentConverter.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace CommunityToolkit.WinUI.Controls;
+
+internal partial class NullToTransparentConverter : IValueConverter
```

Comment:
Hmm, I just noticed our other converters here are public (and also pollute the controls namespace...)

Not much we can do to fix that now, but this being internal means that folks wouldn't be able to re-template the control.

Reply:
Sorry. So we can't fix this binding issue now?

Reply:
@Poker-sang sorry for the confusion. I just mean, please make this `public`. :)
```suggestion
public partial class NullToTransparentConverter : IValueConverter
```
I can apply this and we can merge.

I was just commenting that we missed that these were public and in the root namespace when we migrated this control into the new codebase (in its original PR); but since we shipped it already in 8.1 it wouldn't be good to change it (as breaking change) even if these are really meant to be more internal to this control. Generally, we'd have them in a namespace underneath controls so they're not visible to consumers of the Toolkit like they would be now.

@Arlodotexe we should have a tracking item for fixing these and maybe other ones that may have leaked. I don't think we just want to break it in 8.2, though maybe we could as they aren't supposed to be used by others anyway...?